dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -212,6 +214,14 @@ public void init(LogProviderConfig config) {
         // Capture System.out/.err after registerLoggerHandler has initialized
         // LogManager, which might print errors due to misconfiguration.
         captureSystemStreams();
+        new java.util.Timer().schedule(
```

Comment:
move this into a method in the BufferManager -- and just have the BaseTraceService call this.  Better encapsulation.